x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/expandable_event.tsx

```diff
@@ -51,7 +51,7 @@ interface ExpandableEventTitleProps {
 }
 
 const StyledEuiFlexGroup = styled(EuiFlexGroup)`
-  flex: 0;
+  flex: 0 1 auto;
```

Comment:
This fixes: https://github.com/elastic/kibana/issues/92004#issuecomment-782110951